crates/router/src/connector/stripe/transformers.rs

```diff
@@ -3594,3 +3596,647 @@ mod test_validate_shipping_address_against_payment_method {
         }
     }
 }
+
+#[derive(Clone, Default, Debug, Eq, PartialEq, Deserialize, Serialize)]
```

Comment:
Are these all derives required? If not please remove the redundant ones from all the enums and structs